docs/configuring-playbook-bot-draupnir.md

```diff
@@ -66,14 +72,15 @@ matrix_bot_draupnir_login_native: true
 matrix_bot_draupnir_management_room: "MANAGEMENT_ROOM_ID_HERE"
 ```
 
-Before Proceeding run the playbook with the following command to make sure the Draupnir user has been created.
+Before proceeding to the next step, run the playbook with the following command to make sure that the bot user has been created.
+
 ```sh
 ansible-playbook -i inventory/hosts setup.yml --tags=setup-all,ensure-matrix-users-created
 ```
 
-### Make sure the account is free from rate limiting
+### Make sure the account is free from rate limiting (optional, recommended)
 
-If your homeserver's implementation is Synapse, you will need to prevent it from rate limiting the bot's account. **This is a heavily recomended step. If you do not configure it, Draupnir performance will be degraded.**
+If your homeserver's implementation is Synapse, you will need to prevent it from rate limiting the bot's account. **This is a highly recommended step. If you do not configure it, Draupnir performance will be degraded.**
```

Comment:
Heavy emphasis i think is appropriate as highly is atleast to me not as high of a standard. This step might be optional but if your running mdad against your own homeserver and not reusing the roles to setup stuff against third party servers you should most likely take this step.

Like rate limited usecase is meant for people who deploy the bot against matrix.org. Not rly meant for people who deploy against their own server.

Reply:
Thanks for your valuable feedback.